psycop/common/model_training_v2/config/config_utils.py

```diff
@@ -0,0 +1,20 @@
+from pathlib import Path
+
+from confection import Config
+
+from psycop.common.model_training_v2.config.baseline_registry import BaselineRegistry
+from psycop.common.model_training_v2.config.baseline_schema import (
+    BaselineSchema,
+)
+from psycop.common.model_training_v2.config.populate_registry import (
+    populate_baseline_registry,
+)
+
+populate_baseline_registry()
```

Comment:
Populating the registry right before load seems to be an OK way of doing it, @KennethEnevoldsen?